pkg/client/redhatsso/config.go

```diff
@@ -0,0 +1,55 @@
+package redhatsso
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared"
+	"github.com/spf13/pflag"
+)
+
+type RedhatSSOConfig struct {
+	BaseURL    string      `json:"base_url"`
+	KafkaRealm RealmConfig `json:"redhatsso_realm"`
+}
+
+type RealmConfig struct {
+	Realm            string `json:"realm"`
+	ClientID         string `json:"client-id"`
+	ClientIDFile     string `json:"client-id_file"`
+	ClientSecret     string `json:"client-secret"`
+	ClientSecretFile string `json:"client-secret_file"`
+	TokenEndpointURI string `json:"token_endpoint_uri"`
+	APIEndpointURI   string `json:"api_endpoint_uri"`
+}
+
+func (c *RealmConfig) setDefaultURIs(baseURL string) {
+	c.TokenEndpointURI = baseURL + "/auth/realms/" + c.Realm + "/protocol/openid-connect/token"
+	c.APIEndpointURI = baseURL + "/auth/realms/" + c.Realm
+}
+
+func NewRedhatSSOConfig() *RedhatSSOConfig {
+	cfg := &RedhatSSOConfig{
+		KafkaRealm: RealmConfig{
+			ClientIDFile:     "secrets/redhatsso-service.clientId",
+			ClientSecretFile: "secrets/redhatsso-service.clientSecret",
+		},
```

Comment:
realm default value will be "redhat-external"

Reply:
done